src/components/Footer.js

```diff
@@ -0,0 +1,56 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+export class Footer extends Component {
+    render() {
+        return (
+            <footer>
+                <div className='top-companies'>
+                    <h2>Top companies choose <a href="">Udemy Business</a> to build in-demand career skills.</h2>
+                    <div className='companies-logos'>
+                        <img alt='Nasdaq' src='https://s.udemycdn.com/partner-logos/v4/nasdaq-light.svg' />
```

Comment:
alt text must be descriptive and i said that many times before

Reply:
this is an image of the logo of a company called "Nasdaq", so i used the company's name as the alt text.. what should i use instead?